aries-site/src/pages/templates/side-to-side-navigation.mdx

```diff
@@ -1,3 +1,101 @@
-import { ComingSoon } from '../../components';
+import { Box, Image, NameValueList, NameValuePair } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
 
-<ComingSoon />
\ No newline at end of file
+## When to use
+
+- Tabs to maneuver between relevant information within the same view.
+- Hamburger menus, sidebars, nav bars, wizards.
+- Switching from one service to another service.
+- Sidebar to navigate between pages.
+- Accordions to open a single panel to navigating between content within the same page.
+
+## HPE Local Header Example
+
+The HPE product’s local header which contains grouped, frequently used and top-level information.
+
+Example coming soon.
```

Comment:
wonder if we should make this bold to let the user know by glancing looks like content

Reply:
Good call, bolded.